test/interactions/interactionTests.ts

```diff
@@ -0,0 +1,110 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Interactions", () => {
+  describe("Interaction", () => {
```

Comment:
It's a convenience. If you want to run just the tests in this file, you can click on the `Interaction` in the tests interface. (I very often want to do that). If you put it at the root, there is no way to do it